apps/datahub/src/app/home/home-header/home-header.component.ts

```diff
@@ -84,9 +84,7 @@ export class HomeHeaderComponent {
         customSearchParameters.filters
       )
     )
-    this.searchService.setSortAndFilters(
-      searchFilters,
-      sortByFromString(customSearchParameters.sort)
-    )
+    const sortBy = await firstValueFrom(this.searchFacade.sortBy$)
```

Comment:
Could e we make the `sort` parameter in the TOML configuration at least optional? not being able to specify a sort order sounds quite limiting for this feature

Reply:
Sure, good idea :slightly_smiling_face: